site-src/api-types/gateway.md

```diff
@@ -1,5 +1,9 @@
 # Gateway
 
+!!! info "Feature State"
+
+    The `Gateway` resource is available in `Beta` as of `v0.5.0`
```

Comment:
Maybe something like this, not quite sure how this will render, but hoping this will result in an expandable version that takes up a bit less space by default (https://squidfunk.github.io/mkdocs-material/reference/admonitions/).

```suggestion
??? success "Standard Channel in v0.5.0+"

    The `Gateway` resource is Beta and part of the Standard Channel in `v0.5.0+`.
```

Reply:
Done